### PR TITLE
uxpt(pb-4814): Updated the list component with indentation and list s…

### PR DESCRIPTION
**Pull request by sangeethaathiyursridharan**
**Normal List**
image

**Ordered List**
image

**Unordered List**
image

**Structured Roman List**
image

**Structured Decimal List**
image

**Structured Bullet List**
image

### CODE REVIEW

File: packages/core/src/List/List.tsx [hunk 1]
```diff
@@ -3,6 +3,24 @@ import { fontSize, space, width, compose } from 'styled-system'
 import themeGet from '@styled-system/theme-get'
 import { getPaletteColor } from '../utils'
 
+const tabSpacingSize = {
+  default: '40px',
+  md: '32px',
+  sm: '24px',
+  xsm: '16px',
```
Comment (pcln-james) [inline packages/core/src/List/List.tsx:10]: nb: our other sizes go from smallest to biggest

**Review by pcln-james — APPROVED** [review]

File: packages/core/src/List/List.tsx (hunk 1 above)
Comment (sangeethaathiyursridharan) [reply packages/core/src/List/List.tsx:10]: {
 xsm: '16px',
 sm: '24px'
 md: '32px',
 default: '40px'
 }
 
 like this.?

Comment (pcln-james) [reply packages/core/src/List/List.tsx:7]: yes exactly 😄

Comment (sangeethaathiyursridharan) [reply packages/core/src/List/List.tsx:7]: i updated it.

**Pull request merged**